www/gatsby-node.js

```diff
@@ -475,6 +487,7 @@ exports.createPages = ({ graphql, actions, reporter }) => {
             node {
               fields {
                 slug
```

Comment:
It might end up causing confusion since here, "slug = path without locale".

Reply:
I guess in the interim phase we have to live with that, once we have all parts of the site we could make a `localizedSlug` or localize the `slug`?